app/views/checkout/_terms_and_conditions.html.haml

```diff
@@ -0,0 +1,2 @@
+%p.small
```

Comment:
Do not render this empty `p.small` tag if not necessary. You could put the `if` before it :)

Reply:
Hi Andres! Welcome to OFN! It's great to have your input here.
I did move the if above in the follow up PR :+1: 
https://github.com/openfoodfoundation/openfoodnetwork/pull/5994/files#diff-b38e245a26b89f5ee71d78eadee38163R2

The p.small is necessary for formatting.